tests/test_backends.py

```diff
@@ -0,0 +1,41 @@
+import numpy as np
```

Comment:
This shouldn't be included in the PR probably right?

Reply:
No.